doc/developer_guide/psy_data.rst

```diff
@@ -743,10 +783,10 @@ coded in the library, the rest is taken from the base class:
         ProvideFieldVectorDouble, &
         {{all_provides|join(", &\n"+indent) }}
 
-.. note:
-    Using ``or ""`` in the jinja statements avoids that ``None`` is
+.. note::
+    Using ``or ""`` in the Jinja statements avoids having ``None``
```

Comment:
The "``or" bit of this note is not rendering correctly for me (pdf).

Reply:
It looks correct for me, though harder to see in PDF than in html. I am not sure how to improve that. I could use single quotes? Italic? None of which would be really easy to see :( Put it on a separate line??